.node-version

```diff
@@ -1 +1 @@
-v16.17.0
+v18.12.1
```

Comment:
I guess preference. maybe there is a way to provide multiple node versions, i just haven't found it.